src/pages/billing/Project.js

```diff
@@ -452,6 +454,11 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
   useOnMount(() => { reloadBillingProjectUsers() })
 
   useEffect(() => { StateHistory.update({ projectUsers }) }, [projectUsers])
+  // Update cost data only if report date range changes, or if spend report tab was selected.
+  useEffect(() => { maybeLoadTotalCost() },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I had to disable `exhaustive-deps` to stop the warning about calling `maybeLoadTotalCost`. If I add `maybeLoadTotalCost` to the dependencies things break because a lot of state is being used within that method. The point though is that I don't want to fetch the data unless the user has requested a different report length (or the tab was initially selected).